frontend/assets/stylesheets/components/_payment-b.scss

```diff
@@ -173,6 +173,12 @@
       @include mq($from: tablet) {
         display: block;
       }
+
+      a {
+        color: guss-colour(guardian-brand);
+        display: block;
+        margin-top: 10px;
+      }
```

Comment:
Do we still need this as a separate file? The `b` refers to the B part of the AB test that's no longer running doesn't it?